docs/xks/operator-guide/kubernetes/eks.md

```diff
@@ -101,6 +101,133 @@ Run Terraform in the following order:
 Make sure that you only have one peering request open at the same time, else the accepter side won't be able to find a unique request.
 Now you should be able to see the VPC peering connected on both sides.
 
+## Update cluster version
+
+Updating EKS cluster version can not be done by updating Terraform code only, it also involves the AWS CLI and kubectl. Find your EKS version to upgrade to here: [EKS versions](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html)
+
+For further information on the AWS CLI commands used in this section, please refer to [AWS EKS CLI](https://docs.aws.amazon.com/cli/latest/reference/eks/index.html)
+
+### Update the control plane
+
+Get the name of the cluster to update:
+
+```bash
+aws eks list-clusters --region eu-west-1
+```
+
+Update the control plane version by running the following command with the actual values for \<cluster-name\> and \<version>\:
```

Comment:
Is there some command to show the available k8s versions that I can update to?

Reply:
Would have been nice but I didn't find one.